Base/usr/share/man/man5/pp.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+pp - Pixel Paint application file format (.pp)
+
+## Description
+
+Pixel Paint files are the products of Pixel Paint application.
+They contain all sorts of drawing information that is used by Pixel Paint application. 
+
+Each file contains the following information:
+- width
+- height
+- layers (optional)
+  - width
+  - height
+  - name
+  - locationx
+  - locationy
+  - opacity_percent
+  - visible
+  - selected
+  - bitmap
+- guides (optional)
+  - offset
+  - orientation
+
+To get .pp file all you need is to start Pixel Paint application and save the file
+
+`File > Save`
+
+Later, you can open this file and Pixel Paint session will start.
+All .pp files launch Pixel Paint application based on PixelPaint.af applicaiton file. 
+
+## See also
+
+-   [ApplicationFile (5)](help://man/5/ApplicationFile)
```

Comment:
Again; this link is broken and not following manpage name format conventions.

Reply:
Done.